Dockerfile

```diff
@@ -1,14 +1,22 @@
 #using neurodebian runtime as parent image
-FROM neurodebian:xenial-non-free
+FROM neurodebian:bionic-non-free
 MAINTAINER The C-PAC Team <cnl@childmind.org>
 
+ARG DEBIAN_FRONTEND=noninteractive
+
 RUN apt-get update
 
 # Install the validator
-RUN apt-get install -y curl && \
-     curl -sL https://deb.nodesource.com/setup_11.x | bash - && \
-     apt-get install -y nodejs
-RUN npm install -g bids-validator
+RUN apt-get install -y apt-utils curl && \
+     curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
```

Comment:
Can't the same effect be achieved by `curl https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash`? `curl` sends its output to stdout by default.

Reply:
Resolved in b5b1714